src/login/mobile-token-onboarding/NoMobileToken.tsx

```diff
@@ -0,0 +1,71 @@
+import {useTranslation} from '@atb/translations';
+import {useNavigation} from '@react-navigation/native';
+import {View} from 'react-native';
+import ThemeText from '@atb/components/text';
+import MobileTokenOnboardingTexts from '@atb/translations/screens/subscreens/MobileTokenOnboarding';
+import Button from '@atb/components/button';
+import {StyleSheet} from '@atb/theme';
+import React from 'react';
+import {StaticColorByType} from '@atb/theme/colors';
+import {CrashSmall} from '@atb/assets/svg/color/images';
+import {settingToRouteName} from '@atb/utils/navigation';
+import {usePreferenceItems} from '@atb/preferences';
+import {useAppState} from '@atb/AppContext';
+
+const themeColor: StaticColorByType<'background'> = 'background_accent_0';
+
+export function NoMobileToken({}: {}): JSX.Element {
+  const styles = useThemeStyles();
+  const {t} = useTranslation();
+  const navigation = useNavigation();
```

Comment:
As much as possible we avoid using `useNavigation` as we then don't have type safety regarding the stack you are in, and as such no type safety on the `navigation.navigate` parameter. Here (and in the other onboarding screens) you could rather follow the pattern done in the `WelcomeScreen` in the normal onboarding:
```ts
export type WelcomeScreenProps = {
  navigation: StackNavigationProp<OnboardingStackParams>;
};

export const WelcomeScreenLogin = ({navigation}: WelcomeScreenProps) => {
```